src/constants.js

```diff
@@ -118,7 +118,7 @@ export const PLUGIN_NAMES = {
   PROVIDER: 'provider',
   WIDGET: 'widget',
   LOGIN_BUTTON: 'loginButton',
-  CONNECT: 'connect',
+  COMPOSE: 'compose',
```

Comment:
Can we map plugins list in one place? We also have some plugins definitions for `rollup`, but we should manage new ones in multiple places each time.

Reply:
not sure understand, what do you mean `plugins definitions for rollup`?

Reply:
`plugins.json`

Reply:
have a talk with @kazurus and resolve, that we don't need to combine `plugins.json` (used in rollup config) and other plugin definitions (in constants.js)